Update for signac 2.0 and general cleanup (#183)
glotzerlab/signac-docs: docs/source/query.rst

    {"p": {"$gt": 2}}

> Note that we have replaced the value for p with the expression ``{'$gt': 2}`` to select all jobs withe p values greater than 2.

Reviewer: ```suggestion
Note that we have replaced the value for p with the expression ``{'$gt': 2}`` to select all jobs with p values greater than 2.
```

It is possible to use search expressions directly on the command line, for example in combination with the ``$ signac find`` command.
In this case filter arguments are expected to be provided as valid JSON expressions.

> However, for simple filters you can also use a simplified syntax in lieu of writing JSON.

Reviewer: Simplify this wording.
```suggestion
For simple filters, you can use a simplified syntax instead of writing JSON.
```
